src/Core/Framework/Plugin/Command/MakerCommand.php

```diff
@@ -100,19 +100,48 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $directory
             );
 
+            if (\method_exists($this->generator, 'disableAskConfirmQuestion')) {
+                $this->generator->disableAskConfirmQuestion();
+            }
+
             $this->generator->addScaffoldConfig($configuration, $input, $io);
 
-            $stubCollection = $this->scaffoldingCollector->collect($configuration);
+            $stubCollection = new StubCollection();
+            $this->generator->generateStubs($configuration, $stubCollection);
+
+            $configsStubs = $this->removeConfigsFromMainCollection($stubCollection);
 
             $this->scaffoldingWriter->write($stubCollection, $configuration);
 
             $io->success('Scaffold created successfully');
 
+            foreach ($configsStubs as $stub) {
```

Comment:
So now we never write the config files even if they don't exist? Honestly I don't think this is a good dev experience. Imo, if it exists we should write it, if not show the new content that needs to be manually applied. And I guess it's the same for any file, we don't need to distinguish between config files with this path matching you have?